spring-cloud-kubernetes-ribbon/pom.xml

```diff
@@ -120,6 +120,7 @@
 		<dependency>
 			<groupId>io.fabric8</groupId>
 			<artifactId>mockwebserver</artifactId>
+			<version>0.0.13</version>
```

Comment:
Since, we do have a central place for keeping versions and properties, I'd see it needs to go there.